src/main/java/net/dv8tion/jda/api/entities/Activity.java

```diff
@@ -251,7 +251,7 @@ static Activity of(@Nonnull ActivityType type, @Nonnull String name, @Nullable S
      */
     static boolean isValidStreamingUrl(@Nullable String url)
     {
-        return url != null && url.matches("https?://(www\\.)?twitch\\.tv/.+");
+        return url != null && url.matches("https?://(www\\.)?(twitch\\.tv/|youtube.com/watch\\?v=).+");
```

Comment:
I think `https://youtu.be/code` is also a valid url for videos/livestreams, isn't it?

Reply:
thats a url-shortener... and discord will probably not use it

Reply:
I tested and this url format doesn't work.